compiler/plugins/target/AMD-AIE/air/CMakeLists.txt

```diff
@@ -151,24 +175,42 @@ iree_cc_library(
     "${IREE_MLIR_AIR_SOURCE_DIR}/include/air/Conversion/Passes.h"
   SRCS
     "ConversionPasses.cpp"
+    "${IREE_MLIR_AIR_SOURCE_DIR}/lib/Conversion/ConvertToAIRPass.cpp"
+    "${IREE_MLIR_AIR_SOURCE_DIR}/lib/Conversion/AIRLoweringPass.cpp"
```

Comment:
AIR-to-AIRRt lowering. This pass and the airrt-to-ipu pass together generates the IPU instructions. This method for generating IPU instrs is our current method to get the IPU laptop examples working.